reviewers/GrammarChecker.js

```diff
@@ -81,10 +96,10 @@ checker.prototype = {
         while ( result = findQuotedStrings.exec( line ) ) {
           nothingToDo = false;
           
```

Comment:
Don't make functions within a loop
```javascript
          (function GrammarCheckerStepClosure( composedLine ) {
```
'composedLine' is already declared in the upper scope.
```javascript
          (function GrammarCheckerStepClosure( composedLine ) {
```